### PR TITLE
Remove warning message when CDI 4.0 option used on lower versions

### LINKED ISSUE #21094: Remove warning if CDI 4.0 specific config option is used with lower versions of CDI.
Remove `CWOWB1016W`.

Since the config option will not have any effect, there is no need to provide a warning.

### PR DESCRIPTION
**Pull request by tevans78**
fixes #21094 

#build

### CODE REVIEW

File: dev/com.ibm.ws.cdi.beansxml_fat/fat/src/com/ibm/ws/cdi/beansxml/fat/tests/CDIConfigTest.java [hunk 1]
```diff
@@ -87,7 +86,13 @@ public void testCdi12Warning() throws Exception {
     }
 
     @Test
-    @SkipForRepeat(EERepeatActions.EE10_ID)
+    /**
+     * Initially we were going to output a warning if emptyBeansXMLExplicitBeanArchive was set when using a CDI version less than 4.0.
```
Comment (Emily-Jiang) [inline dev/com.ibm.ws.cdi.beansxml_fat/fat/src/com/ibm/ws/cdi/beansxml/fat/tests/CDIConfigTest.java]: I suggest not to add this in the code. You can add the comments to the issue instead.

File: dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops [hunk 2]
```diff
@@ -93,9 +93,10 @@ cdi12.element.type.superceded.CWOWB1015W=CWOWB1015W: The cdi12 configuration ele
 cdi12.element.type.superceded.CWOWB1015W.explanation=The cdi configuration element replaces the cdi12 configuration element and contains all the same options, plus new options.
 cdi12.element.type.superceded.CWOWB1015W.useraction=Change your server.xml configuration to use the cdi configuration element instead of the cdi12 configuration element.
 
-cdiEmptyBeansXMLExplicitBeanArchive.ignored.CWOWB1016W=CWOWB1016W: The emptyBeansXMLExplicitBeanArchive attribute of the cdi configuration element is supported only on CDI 4.0 or newer. This attribute is ignored.
-cdiEmptyBeansXMLExplicitBeanArchive.ignored.CWOWB1016W.explanation=The emptyBeansXMLExplicitBeanArchive attribute has no effect when it is used with a CDI version lower than 4.0.
-cdiEmptyBeansXMLExplicitBeanArchive.ignored.CWOWB1016W.useraction=Update your server.xml configuration to either remove the attribute or use CDI version 4.0 or newer.
+#CWOWB1016W is not used. The emptyBeansXMLExplicitBeanArchive config option is silently ignored if not running CDI 4.0+
```
Comment (Emily-Jiang) [inline dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops]: don't comment out. Just delete them.

Comment (tevans78) [reply dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops]: Need to ensure that the code is not reused so want to leave something here

File: dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops [hunk 3]
```diff
@@ -93,10 +93,7 @@ cdi12.element.type.superceded.CWOWB1015W=CWOWB1015W: The cdi12 configuration ele
 cdi12.element.type.superceded.CWOWB1015W.explanation=The cdi configuration element replaces the cdi12 configuration element and contains all the same options, plus new options.
 cdi12.element.type.superceded.CWOWB1015W.useraction=Change your server.xml configuration to use the cdi configuration element instead of the cdi12 configuration element.
 
-#CWOWB1016W is not used. The emptyBeansXMLExplicitBeanArchive config option is silently ignored if not running CDI 4.0+
-#cdiEmptyBeansXMLExplicitBeanArchive.ignored.CWOWB1016W=CWOWB1016W: The emptyBeansXMLExplicitBeanArchive attribute of the cdi configuration element is supported only on CDI 4.0 or newer. This attribute is ignored.
-#cdiEmptyBeansXMLExplicitBeanArchive.ignored.CWOWB1016W.explanation=The emptyBeansXMLExplicitBeanArchive attribute has no effect when it is used with a CDI version lower than 4.0.
-#cdiEmptyBeansXMLExplicitBeanArchive.ignored.CWOWB1016W.useraction=Update your server.xml configuration to either remove the attribute or use CDI version 4.0 or newer.
+#CWOWB1016W is not used - https://github.com/OpenLiberty/open-liberty/pull/21107
```
Comment (Emily-Jiang) [inline dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops:96]: Why do you have to mention this? I would just remove it. If you want, add this to the issue itself not in this file.

**Review by Emily-Jiang — APPROVED** [review]
LGTM!

**Comment by tevans78** [comment]
#libby

**Pull request merged**